packages/ra-ui-materialui/src/form/checkForInitialValues.js

```diff
@@ -0,0 +1,22 @@
+import merge from 'lodash/merge';
+
+const checkForInitialValues = (initialValues, newValues) => {
```

Comment:
by reading that method name, I don't understand what it's supposed to do or to return. Can you add jsDoc, example usage, or change the name?